scripts/extension-version.sh

```diff
@@ -3,9 +3,16 @@
 set -eu
 
 CHANNEL="$1"
+NEXT="$2"
 
 if [ "$CHANNEL" = "dev" ]; then
-    vsce show Prisma.prisma-dev --json | jq ".versions[0].version" | tr -d '"'
+    EXTENSION_VERSION=$(vsce show Prisma.prisma-dev --json | jq ".versions[0].version" | tr -d '"')
```

Comment:
We rely on `vsce show` to get the current extension version and increment it to get the next extension version.

But marketplace is not as fast as npm registery and can take more than 5 minutes to register/verify a new release. 

This might fail and produce colliding next extension versions, if we are releasing alphas too quickly. 

However, that should be okay, as it will be fixed in the next alpha release automatically. 

We can try to circumvent this by trying to store the next extension version like we do for prisma version but that has the following downsides: 

1. Makes management of state more complicated
1. We are not aware what happens when we publish an extension while another extension is still pending publishing on the verification step.

Reply:
This is an example of when this happens - https://github.com/prisma/vscode/runs/625561915?check_suite_focus=true

This one was manually triggered to check.

Reply:
The CI should reconcile this in the next run automatically.

Reply:
Example of CI correcting course: https://github.com/prisma/vscode/runs/625590335?check_suite_focus=true